nestkernel/event_delivery_manager.cpp

```diff
@@ -620,10 +677,30 @@ EventDeliveryManager::gather_target_data( const thread tid )
     kernel().connection_manager.save_source_table_entry_point( tid );
 #pragma omp barrier
     kernel().connection_manager.clean_source_table( tid );
+
+#ifdef TIMER_DETAILED
+#pragma omp barrier
+    if ( tid == 0 )
+    {
+      // TODO_SDR: Following function call commented out because it causes real runs to hang!
+      // kernel().mpi_manager.synchronize(); // to get an accurate time measurement
+      //                                    // across ranks
```

Comment:
Can this be removed?

Reply:
I agree, we shouldn't introduce a MPI synchronization point. This would yield significant overhead, which is also not accounted for in any timer at the moment.

Reply:
Agreed. Now removed synch point and comment.